serializations.md

```diff
@@ -121,6 +141,8 @@ subsequent sections divided by object.
 | Channel settle transaction | 56 |
 | Channel off chain transaction | 57 |
 | Channel | 58 |
+| Key block | 100 |
```

Comment:
Not needed as it only makes sense for serialization as `S(O) = rlp([tag(O), vsn(O)] ++ fields(O))` while serialization is defined as byte array.

Reply:
See also https://github.com/aeternity/epoch/pull/1202#discussion_r191372961 for more context.